### PR TITLE
[Merged by Bors] - chore: scope symmDiff notations

### PR DESCRIPTION
**Pull request by PatrickMassot**
Those notations are not scoped whereas the file is very low in the import hierarchy.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Order/SymmDiff.lean [hunk 1]
```diff
@@ -70,10 +70,12 @@ def bihimp [Inf α] [HImp α] (a b : α) : α :=
 /- This notation might conflict with the Laplacian once we have it. Feel free to put it in locale
   `order` or `symmDiff` if that happens. -/
```
Comment (sgouezel) [inline Mathlib/Order/SymmDiff.lean:71]:
```suggestion
```

**Review by sgouezel — COMMENTED** [review]
bors d+

**Review by YaelDillies — COMMENTED** [review]
Can you scope that in `Order` instead?

**Comment by YaelDillies** [comment]
Also please include motivation for this change. This PR is currently unjustified for someone coming across it on GitHub.

**Comment by PatrickMassot** [comment]
bors merge

**Pull request closed without merging**